### PR TITLE
feat: created Partner, PartnerImages, Partner.mocks, Partner.stories …

### PR DESCRIPTION
**Pull request by mohiandipta**
…in feature

created Partner section into `component/ui/feature` with product image from cms data.

#80 

# Description

created partner component for show title, subtitle and PartnerImages component for showing partnerImg. got type from CMSProduct, Mock data created into Partner.mocks and got product data from utils/fetchProduct

image

### CODE REVIEW

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
you have cooked me very tasty food, I like :+1: 

Do the changes, get back to me

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+WithProduct.args = {
+    title: undefined,
+};
+
+export const WithoutProduct = Template.bind({});
+WithoutProduct.args = {
+    ...PartnerMock,
+    theme: undefined,
+    subTitle: undefined,
+};
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.stories.tsx]: done

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.mocks.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { PartnerProps } from "./Partner";
+
+export const PartnerMock: PartnerProps = {
+    theme:"primary-brown",
+    title: 'Our Popular Products',
+    subTitle: "We're partners with countless major organisations around the globe",
+}
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.mocks.tsx]: done

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Default.args = {
+    ...PartnerMock,
+};
+
+export const WithProduct = Template.bind({});
+WithProduct.args = {
+    title: undefined,
+};
+
+export const WithoutProduct = Template.bind({});
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.stories.tsx:28]: okay

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react'
+import { fetchProducts } from '../../../../utils/fetchProducts'
+import PartnerImages from './PartnerImages'
+import { CmsProducts } from '~/types/types'
+import { Grid } from '@mui/material'
+import { ThemeName } from 'ui/components/foundations/theming'
+
+export type PartnerProps = {
+        theme?: ThemeName
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.tsx:9]: yes

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        }
+        fetchData()
+    }, [])
+
+    return (
+        <div className='h-auto w-full text-center bg-white text-black'>
+            <section className="bg-partner relative max-w-full sm:mx-6 my-24 shadow sm:rounded-2xl overflow-hidden">
+                <div className="w-full px-6 sm:px-0 py-20 flex flex-col items-center justify-center space-y-4 text-center">
+                    <h3 className="text-2xl text-neutral-800 font-semibold">{props.title ? "title" : "Our Popular Products"}</h3>
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/Partner.tsx:39]: okay

File: apps/akij-food-beverage/src/components/ui/features/PartnerSection/PartnerImages.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+import { urlForThumbnail } from '../../../../utils/imageProcess'
+
+export type ProductImage = {
+    product: CmsProducts[]
+}
+
+export default function PartnerImages({ product }: ProductImage) {
+    return (
+        <div>
+            <CardMedia className='h-40' component="img" image={urlForThumbnail(product.image)} ></CardMedia>
```
[the remark this answers is not in the document]
Comment (mohiandipta) [reply apps/akij-food-beverage/src/components/ui/features/PartnerSection/PartnerImages.tsx:12]: done

**Pull request merged**